src/js/TextFields/TextField.js

```diff
@@ -175,18 +193,25 @@ export default class TextField extends Component {
     }
 
     const textFieldProps = {
-      ...props,
       value,
-      className: classnames('md-text-field', className, {
+      className: classnames('md-text-field', inputClassName, {
         active,
         'floating-label': useFloatingLabel,
         'single-line': !useFloatingLabel && !multiline,
         'multi-line': multiline,
         'full-width': fullWidth,
       }),
-      onFocus: this.handleFocus,
+      style: inputStyle,
```

Comment:
onFocus was alphabetically sorted in the line 206.

Fixed other issues in the lines 219 and 242.